spec/parliament/request_spec.rb

```diff
@@ -63,9 +63,27 @@
   end
 
   describe '#get' do
```

Comment:
Should we also check our delegation? Ensuring all methods we *think* are being delegated actually are?